common/py/pydantic.py

```diff
@@ -1,80 +1,29 @@
-import random
-from pydantic_ai import Agent, RunContext
 from agentuity import AgentRequest, AgentResponse, AgentContext
+from pydantic_ai import Agent
 
+client = Agent("openai:gpt-4o-mini")
 
 def welcome():
     return {
-        "welcome": "Welcome to the Pydantic Agent! I can help you interact with spinning the roulette wheel.",
+        "welcome": "Welcome to the Pydantic Python Agent! I can help you build AI-powered applications using Pydantic.",
         "prompts": [
             {
-                "data": "Put my money on square eighteen",
-                "contentType": "text/plain",
+                "data": "How do I use Pydantic to call OpenAI models?",
+                "contentType": "text/plain"
             },
             {
-                "data": "I bet five is the winner",
-                "contentType": "text/plain",
-            },
-        ],
+                "data": "What are the best practices for prompt engineering with Pydantic?",
+                "contentType": "text/plain"
+            }
+        ]
     }
 
-
-# Example taken from: https://ai.pydantic.dev/agents/#introduction
-
-# The pydantic agent
-roulette_agent = Agent(
```

Comment:
Another vastly different one that IMO should be an example, not the starter template.